namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/main.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
```

Comment:
This should be a copy of this file:

https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/blob/master/examples/main.tf